web.d.ts

```diff
@@ -0,0 +1,69 @@
+import { Writable } from '.'
+
+export interface ReadableStreamDefaultReader {
+  read(): Promise<unknown>
+  cancel(reason?: Error | string | null): Promise<void>
```

Comment:
```suggestion
  cancel(reason?: unknown): Promise<void>
```

The reason can be anything.